### DISCUSSION TITLE
第2节课笔记(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#169] [笔记]
提交笔记链接

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection2-Easy-Fun-with-InternLM-Entertaining-Demo/#%E7%AC%94%E8%AE%B0

**Comment by yaoyaotaba** [comment]


**Reply by yaoyaotaba** [reply]
https://www.zhihu.com/question/622961031/answer/3357485287

**Comment by PlateAss** [comment]
https://plateass.github.io/class2

**Comment by yayayayaye** [comment]
https://b3x9893p70.feishu.cn/docx/YonAdb25eoz0RgxRxcUc2syjnUh?from=from_copylink

**Comment by Ace-may** [comment]
https://blog.csdn.net/qq_45079433/article/details/135507036

**Comment by Midkey** [comment]
https://blog.csdn.net/qq_24965035/article/details/135559687?spm=1001.2014.3001.5502

**Comment by Midkey** [comment]
https://blog.csdn.net/qq_24965035/article/details/135559687?spm=1001.2014.3001.5502

**Comment by relishLeisure** [comment]
http://t.csdnimg.cn/gUmHs

**Comment by yaoyaotaba** [comment]
https://www.zhihu.com/question/622961031/answer/3357485287

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135761386

**Comment by 1131764933** [comment]
https://shimo.im/file-invite/RrSbDBviNNvqrcqD48wjJBOe3nwq6/ 颜洪斌 邀请您协作《轻松玩转书生·浦语大模型趣味Demo》，点击链接开始编辑
